lib/PlatoonService/src/PlatoonUtils.h

```diff
@@ -67,6 +67,7 @@ namespace PlatoonUtils
 
     /**
      * Calculate the heading at which a target waypoint is located relative to other waypoint.
+     * The result is the heading in mrad. The result range is [-PI, PI].

```

Comment:
Its the question for all other functions with angles too ... what is the expected range.